contracts/staking/TokenStaking.sol

```diff
@@ -941,25 +822,14 @@ contract TokenStaking is Initializable, IStaking, Checkpoints {
                 .authorized;
     }
 
-    /// @notice Returns staked amount of T, Keep and Nu for the specified
-    ///         staking provider.
-    /// @dev    All values are in T denomination
-    function stakes(address stakingProvider)
```

Comment:
I just realized we will be breaking stuff that consumes Threshold staking information by removing this method. For example, here's the dashboard unpacking the result from `stakes()`:
https://github.com/threshold-network/token-dashboard/blob/8264be8e00286f9d51674fd07ca67030a40ed168/src/threshold-ts/staking/index.ts#L220

What do you think about keeping this method, but marking it as deprecated? Still, I'd keep the `stakeAmount()` method. In the meantime, we can identify stuff that can be updated or removed in the dashboard after legacy support was dropped, which I think it's probably a lot.

Reply:
✔️